includes/multi-currency/Analytics.php

```diff
@@ -98,6 +112,34 @@ public function register_admin_scripts() {
 		);
 	}
 
+	/**
+	 * Add the list of currencies used on the store to the wcSettings to allow it to be accessed by the front-end JS script.
+	 *
+	 * @return void
+	 */
+	public function register_customer_currencies() {
+		$currencies           = $this->multi_currency->get_all_customer_currencies();
+		$available_currencies = $this->multi_currency->get_available_currencies();
+		$currency_options     = [];
+
+		foreach ( $currencies as $currency ) {
+			if ( ! isset( $available_currencies[ $currency ] ) ) {
+				continue;
+			}
+
+			$currency_details   = $available_currencies[ $currency ];
+			$currency_options[] = [
+				'label' => $currency_details->get_name(),
```

Comment:
Looks like we will need to decode HTML entities here, I'm seeing `Icelandic kr&oacute;na` for ISK.

Reply:
Great catch! Updated in fa1cbf57.

I also lowered the cache time, because it felt like too much to have 1 hour, especially if new currencies are being used on the store for the first time. I think this is still a good amount, as it prevents the query being made on every page load.